admin/billing/orb.go

```diff
@@ -279,6 +279,35 @@ func (o *Orb) IsInvoicePaid(ctx context.Context, invoice *Invoice) bool {
 	return strings.EqualFold(invoice.Status, "paid")
 }
 
+func (o *Orb) MarkCustomerTaxExempt(ctx context.Context, customerID string) error {
+	// note - hard coded to use Avalara tax provider as of now, Orb supports Avalara and TaxJar
```

Comment:
do we need to make this configurable, if in future we change tax provider, not really urgent now ?

Reply:
I don't know, what do you think?

Reply:
Made it configurable in case we want to disable it (manually managing it) or change at some point in future. 

Configurable through `RILL_ADMIN_ORB_INTEGRATED_TAX_PROVIDER`, default is still `avalara` if not set for convenience as Kristen mentioned that this is going to be our tax provider going forward, can be set to `none` to disable.